03/app.js

```diff
@@ -0,0 +1,15 @@
+const btnsList = document.querySelectorAll('button');
+
+const changeText = function (e) {
+    this.innerText = 'clicked';
+    console.log(this.innerText);
+    if (this.innerText === 'clicked') {
```

Comment:
Czy faktycznie potrzebujemy `if`? Klikamy i od razu usuwamy nasłuchiwanie. Nie musimy tego warunkować. Na jakim elemencie działamy załatwia nam `this`